package.json

```diff
@@ -22,6 +22,7 @@
     "@babel/preset-env": "^7.3.4",
     "async": "^2.0.1",
     "babelify": "^10.0.0",
+    "eth-json-rpc-errors": "github:MetaMask/eth-json-rpc-errors",
```

Comment:
Let's get this published to npm, and added to the MetaMask npm group (will allow npm/yarn to do its appropriate hash-locking).

Reply:
Pushed new commit with [eth-json-rpc-errors](https://www.npmjs.com/package/eth-json-rpc-errors) installed from NPM. I'll leave the adding to the MM org to you.